motion/_animation.scss

```diff
@@ -348,67 +344,51 @@
   from {
     opacity: 1;
     height: 0%;
```

Comment:
Instead of animating height, where you want to squash or expand something but only in one dimension, you can use transform: scaleY. That one is "cheaper" for browser to perform. Please try it out.